aten/src/ATen/templates/RegistrationDeclarations.h

```diff
@@ -0,0 +1,4 @@
+// This file contains all native_functions that can be registered to
+// and the schema string that they should be registered with
```

Comment:
Interesting that you don't need any includes here

Reply:
Aha because we don't include this file directly... There're some additional stuff xla want to set up in the header file about class/namespace so we generate a new file to include. But yea it's possible to make this a real header file for backend extensions.